package/tests/testthat/helper.R

```diff
@@ -0,0 +1 @@
+test_data_path <- testthat::test_path("_data")
```

Comment:
Do we need this and the same in `setup.R`?

Reply:
I think not - that would not be DRY. I move it to helper for interactive debugging. 

> Helper files live in tests/testtthat/, start with helper, and end with .r or .R. They are sourced by devtools::load_all() (so they’re available interactively when developing your packages)

> Setup files live in tests/testthat/, start with setup, and end with .r or .R. Typically there is only one setup file which, by convention, is tests/testthat/setup.R. Setup files are sourced by [test_check()](https://testthat.r-lib.org/reference/test_package.html) and friends (so that they’re available no matter how your tests are executed), but they are not sourced by devtools::load_all().

[source](https://testthat.r-lib.org/articles/special-files.html)